### PR TITLE
Fix a typo in windows.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@fujitsu.com>

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/en/docs/tasks/debug/debug-cluster/windows.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ content_type: concept
 
 1. My pods show status as `ErrImgPull` or `ImagePullBackOff`
 
-   Ensure that your Pod is getting scheduled to a [compatable](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) Windows Node.
+   Ensure that your Pod is getting scheduled to a [compatible](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) Windows Node.
```
Comment (tengqm) [inline content/en/docs/tasks/debug/debug-cluster/windows.md:30]:
```suggestion
   Ensure that your Pod is getting scheduled to a
   [compatible](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
   Windows Node.
```

File: content/en/docs/tasks/debug/debug-cluster/windows.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ content_type: concept
...
 
-   More information on how to specify a compatable node for your Pod can be found in [this guide](/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host).
+   More information on how to specify a compatible node for your Pod can be found in [this guide](/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host).
```
Comment (tengqm) [inline content/en/docs/tasks/debug/debug-cluster/windows.md:32]:
```suggestion
   More information on how to specify a compatible node for your Pod can be found in
   [this guide](/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host).
```

**Comment by fenggw-fnst** [comment]
Thanks for the review, done.

**Pull request merged**